routes/components/employee_row.jsx

```diff
@@ -19,10 +19,30 @@ class EmployeeRow extends React.Component {
         return returnArr;
     }
 
+    renderPosition() {
+        let pos = this.props.position;
+
+        switch (pos) {
+            case 1:
+                return <span className="player-position top-three">🥇</span>
+            case 2:
+                return <span className="player-position top-three">🥈</span>
+            case 3:
+                return <span className="player-position top-three">🥉</span>
+            default:
+                return <span className="player-position">{this.props.position}</span>;
```

Comment:
I'd rewrite this so that you don't, ie only append the "top-three" class when needed and so on.
Define the stuff first and then return the html in the end with the variables